pkg/operator/ceph/cluster/cluster.go

```diff
@@ -634,7 +634,7 @@ func (c *cluster) configureMsgr2() error {
 
 		// set default rbd map options to enable msgr2 in the kernel if it's
 		// required even with encryption disabled
-		if c.Spec.Network.Connections != nil && c.Spec.Network.Connections.RequireMsgr2 {
+		if c.Spec.Network.Connections != nil && c.Spec.RequireMsgr2() {
```

Comment:
The previous conditional judgment can be removed, so it can be changed to `c.Spec.RequireMsgr2()`